src/__init__.py_base

```diff
@@ -0,0 +1,11 @@
+import os
+
+import py_api_raml
```

Comment:
We called the PHP package `elife/api`, so maybe we should aim for `elife_api`?

Reply:
Fine with the name change.

I did try putting it in the dist (it does work from there) but the only issue was that I was to forced to then import it as:
`from dist import schema_directory`

As I could not rename the actual 'dist' directory to which the module name is pulled from I added that a python specific directory be generated during the build. But yes it would nicer to not have copied files, but I think having to `from dist import ..` is too confusing.

Reply:
Downside is that the repository will contain 3 copies (`src`, `dist` and `py`) of schemas, which can be also confusing when using grep or searches. Is it possible to commit the `py_api_raml` module and doing magic inside it (realpath and `../`) to get to the `dist/` folder?

Reply:
OK I have got around it. Best of both worlds I believe, the module `elife_api` will now only contain its `__init__.py` and I just changed the `include` path in the `MAINFEST.in` to be `include dist/model/*.json`. This way the JSON files are not duplicated here, just pulled into the egg when someone installs the package.